js/style/style_function.js

```diff
@@ -8,7 +8,9 @@ exports.interpolated = function(parameters) {
         return inner(globalProperties && globalProperties.zoom, featureProperties || {});
     };
     outer.isFeatureConstant = inner.isFeatureConstant;
-    outer.isGlobalConstant = inner.isGlobalConstant;
+    outer.isZoomConstant = inner.isZoomConstant;
+    outer.getStopZoomLevels = inner.getStopZoomLevels;
```

Comment:
[`parseColor` knows about the syntax of functions](https://github.com/mapbox/mapbox-gl-js/blob/api-introduction-2461/js/style/parse_color.js#L18-L21)

When I'm trying to decide what goes into `mapbox-gl-js` and what goes into `mapbox-gl-function`, I ask myself if this is functionality that another downstream consumer, like Studio, would need to evaluate the function. The `getStopZoomLevels` method feels weirdly granular and mapbox-gl-js-specific. 

If you want to leave this as-is, I respect that decision.